cmd/zdb/zdb.c

```diff
@@ -7527,8 +7539,15 @@ dump_mos_leaks(spa_t *spa)
 	mos_obj_refd(spa->spa_feat_for_read_obj);
 	mos_obj_refd(spa->spa_feat_for_write_obj);
 	mos_obj_refd(spa->spa_history);
-	mos_obj_refd(spa->spa_errlog_last);
-	mos_obj_refd(spa->spa_errlog_scrub);
+
+	if (!spa_feature_is_enabled(spa, SPA_FEATURE_HEAD_ERRLOG)) {
+		mos_obj_refd(spa->spa_errlog_last);
+		mos_obj_refd(spa->spa_errlog_scrub);
+	} else {
+		traverse_errorlog(mos, spa->spa_errlog_last);
```

Comment:
don't we need to mark the `spa_errlog_last` object itself as refd?  i.e. this should look like

```
mos_obj_refd(spa->spa_errlog_last);
mos_obj_refd(spa->spa_errlog_scrub);

if (spa_feature_is_enabled(spa, SPA_FEATURE_HEAD_ERRLOG)) {
 	traverse_errorlog(mos, spa->spa_errlog_last);
	traverse_errorlog(mos, spa->spa_errlog_scrub);
}
```

Reply:
you are correct, will fix.